src/App.tsx

```diff
@@ -93,6 +93,33 @@ const countryToFlagMapping: {[key: string]: string} = {
   Zambia: 'zm',
 };
 
+const PlayerPosition = ({position}: {position: string}) => {
+  let color: string;
+  switch (position) {
+    case 'FW':
+      color = 'success';
+      break;
+    case 'MF':
+      color = 'warning';
+      break;
+    case 'DF':
+      color = 'danger';
```

Comment:
Using 'danger' doesn't sit entirely right with me because it also looks like it could mean "bad" or "injured", but that might be overthinking it.

I immediately thought that 'info' might be a bit circus-tenty, but it's growing on me:
<img width="574" alt="image" src="https://github.com/stianjensen/sheplays-browser/assets/5347151/5864366a-3016-40e8-ba07-c7efe7198431">

Reply:
I think I will also try to make a switch for looking at the teams "in formation", i.e. sorted by position instead of by "selection order", which will hopefully look less circus-tenty